types/Decimal.php

```diff
@@ -99,7 +99,7 @@ public function renderValue($value, \Doku_Renderer $R, $mode)
     public function validate($rawvalue)
     {
         $rawvalue = parent::validate($rawvalue);
-        $rawvalue = str_replace(',', '.', $rawvalue); // we accept both
+        $rawvalue = trim((float)str_replace(',', '.', $rawvalue), 0); // we accept both
 
         if ((string)$rawvalue != (string)(float) $rawvalue) {
             throw new ValidationException('Decimal needed');
```

Comment:
I am wondering. Shouldn't this already prevent the issue you're trying to fix?

Reply:
Thanks for your comments.

I updated the code. Let me know if you have any comments.